### PR TITLE
Add docker development flow to docs

### PR DESCRIPTION
**Pull request by DanielMSchmidt**
**Checklist**
- [ ] Did you add a JIRA issue in a commit message or as part of the branch name?
- [ ] Did you add new unit tests?
- [ ] Did you add new integration tests?
- [ ] If this is a regression, did you write a test to catch this in the future?

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -45,6 +50,16 @@ Node 4.4.x and NPM 3.9.x (and above) are **required**. We suggest using [nvm](ht
 
 6. Navigate to [http://localhost:4200](http://localhost:4200)
 
+### Development with Docker
+
+To start a development server run
+
+```sh
+docker-compose up -d
+docker-compose toolchain exec /bin/bash # This opens bash inside of the docker container
```
Comment (Fabs) [inline README.md]: docker-compose exec toolchain /bin/bash

**Comment by DanielMSchmidt** [comment]
Thank you @Fabs

**Comment by nLight** [comment]
run integration tests

**Comment by nLight** [comment]
run system tests

File: README.md [hunk 2]
```diff
@@ -45,6 +50,16 @@ Node 4.4.x and NPM 3.9.x (and above) are **required**. We suggest using [nvm](ht
 
 6. Navigate to [http://localhost:4200](http://localhost:4200)
 
+### Development with Docker
+
+To start a development server run
+
+```sh
+docker-compose up -d
+docker-compose exec toolchain /bin/bash # This opens bash inside of the docker container
```
Comment (weblancaster) [inline README.md]: Name won't be `toolchain`, if it's the first DCOS-UI container will be `projectName_serviceName_#` so most likely `dcosui_toolchain_1`

Comment (DanielMSchmidt) [reply README.md]: I get where you are coming from and with docker exec you are 💯 right. With docker-compose exec you specify the service name, that's why this command works

**Comment by nLight** [comment]
run system tests

**Pull request merged**